climakitae/explore/warming.py

```diff
@@ -696,6 +632,183 @@ def warming_levels_select(self):
+                """
+                return data.loc[
+                    data == data.quantile(0.5, "all_sims", method="nearest")
+                ].all_sims.values.item()
+
+            def find_sim(all_plot_data, area_avgs, stat_funcs, my_func):
+                if my_func == "Median":
+                    one_sim = all_plot_data.sel(all_sims=stat_funcs[my_func](area_avgs))
+                else:
+                    which_sim = area_avgs.reduce(stat_funcs[my_func], dim="all_sims")
+                    one_sim = all_plot_data.isel(all_sims=which_sim.values)
+                one_sim.all_sims.values = get_name(
+                    one_sim.all_sims.values.item(),
+                    my_func,
+                )
+                return one_sim
+
+            area_avgs = area_average(all_plot_data)
+            stat_funcs = {
+                "Minimum": np.argmin,
+                "Maximum": np.argmax,
+                "Median": arg_median,
+            }
+            stats = xr.concat(
+                [
+                    find_sim(all_plot_data, area_avgs, stat_funcs, one_func)
+                    for one_func in stat_funcs
+                ],
+                dim="all_sims",
+            )
+            stats.name = "Cross-simulation Statistics"
+
+            # Set up plotting arguments
+            width = 410
+            height = 210
+            clabel = (
+                wl_viz.wl_params.variable
+                + " ("
+                + wl_viz.gwl_snapshots.attrs["units"]
+                + ")"
+            )
+            vmin = wl_viz.mins.sel(warming_level=warmlevel).values.item()
+            vmax = wl_viz.maxs.sel(warming_level=warmlevel).values.item()
+            if (vmin < 0) and (vmax > 0):
+                sopt = True
+            else:
+                sopt = None
+
+            # Make plots
+            plot_list = []
+            for stat in stats:
+                plot_list.append(
+                    stat.squeeze()
+                    .drop(["warming_level"])
+                    .hvplot.image(
+                        clabel=clabel,
+                        cmap=wl_viz.cmap,
+                        clim=(vmin, vmax),
+                        symmetric=sopt,
+                        width=width,
+                        height=height,
+                        xaxis=False,
+                        yaxis=False,
+                        title=stat.all_sims.values.item(),  # dim has been overwritten with nicer title
+                    )
+                )
+            all_plots = plot_list[0] + plot_list[1] + plot_list[2]
+
+            all_plots.opts(
+                title=wl_viz.wl_params.variable
+                + ": for "
+                + str(warmlevel)
+                + "°C Warming Across Models"
+            )  # Add title
+            warm_level_dict[warmlevel] = all_plots.cols(1)
+        # else:
+        # return None
+    return warm_level_dict
+
+
```

Comment:
In `warming_levels_visualize` spaces need to be added to the postage_stamps_MAIN static text at the beginning of each line excluding the first. This is why there's a few typos in here.